client/scripts/views/pages/delegate.ts

```diff
@@ -73,11 +72,11 @@ const getDelegate = async (vnode: m.Vnode<{}, IDelegateFormState>) => {
 const setDelegate = async (vnode: m.Vnode<{}, IDelegateFormState>) => {
   if (app.chain.apiInitialized) {
     let delegationPromise: Promise<any>;
-    if (CompoundTypes.EventChains.find((c) => c === app.chain.network)) {
+    if (app.chain.network === ChainNetwork.Compound) {
       delegationPromise = (app.chain as Compound).chain.setDelegate(
         vnode.state.form.address, vnode.state.form.amount
       );
-    } else if (AaveTypes.EventChains.find((c) => c === app.chain.network)) {
+    } else if (app.chain.network === ChainNetwork.Aave) {
```

Comment:
I'm a little concerned calling Compound and Aave "ChainNetworks" too much longer. Really they're just standards for a governance controller that needs to be turned on, using the ChainBase to figure out the provider (currently all of these 'networks' are only supported on Ethereum). I'm not sure on the naming convention just yet, but something to think about soon.

Reply:
We should also discuss enums/object and string naming and iron out where/how that will be used.